lib/driver.js

```diff
@@ -116,6 +116,9 @@ export class Driver {
         this.log("Connecting handler");
         const pr = this.handler.connect();
 
+        if (!await this.subscribe())
+            return false;
```

Comment:
This should be allowed to fail; we might not have addresses yet. In fact I don't think it needs awaiting at all, it can just run fire-and-forget.